.devcontainer/Dockerfile

```diff
@@ -2,20 +2,4 @@
 
 FROM mcr.microsoft.com/vscode/devcontainers/universal:0-linux
 
-USER root
-
-# [Option] Install Docker CLI
-ARG INSTALL_DOCKER="false"
-COPY library-scripts/docker-debian.sh /tmp/library-scripts/
-RUN if [ "${INSTALL_DOCKER}" = "true" ]; then \
-        rm -f /usr/local/share/docker-init.sh \
-        && bash /tmp/library-scripts/docker-debian.sh "true" "/var/run/docker-host.sock" "/var/run/docker.sock" "codespace"; \
-    fi \
-    && rm -rf /var/lib/apt/lists/* /tmp/library-scripts/
-
-USER codespace
```

Comment:
Why deleting this? is it already the case that the USER is implicitly set to `codespace`?

Reply:
Yup exactly! the USER is being set to `codespace` in the base image:

https://github.com/microsoft/vscode-dev-containers/blob/master/containers/codespaces-linux/.devcontainer/base.Dockerfile#L7

https://github.com/microsoft/vscode-dev-containers/blob/master/containers/codespaces-linux/.devcontainer/base.Dockerfile#L121